app/javascript/mastodon/components/autosuggest_textarea.js

```diff
@@ -79,6 +79,7 @@ class AutosuggestTextarea extends ImmutablePureComponent {
     }
 
     // auto-resize textarea
+    e.target.style.height = 'auto';
```

Comment:
Interesting. In the future I would like to consider some form of declarative (state+props=>render) way of handling textarea height, instead of using these side effects and direct DOM manipulation.